spec/features/quotes/show_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Quotes | Index', :vcr do
+  let!(:before) do
+    visit root_path
+    fill_in :keyword, with: 'life happiness'
+    click_on 'Search for Quotes'
+  end
+  describe 'When I enter a key word, I am redirected to the /quotes index and;' do
+    context 'Happy Path' do
+      it 'I see the total amount of quotes found' do
+        expect(page).to have_content 'Total Quotes Found: 243'
+      end
+
+      it 'I see only 10 quotes' do
+        quotes = find_all('.quote')
+        expect(quotes.count).to eq 10
+      end
+
+      it 'I see the correct quote response' do
+        within('#id-NZx4vMwOCDxJ') do
+          expect(page).to have_content 'There is only one happiness in life, to love and be loved.'
+          expect(page).to have_content 'George Sand'
+          expect(page).to have_content 'famous-quotes'
+        end
```

Comment:
I mentioned earlier how the id in your poro can help you test. I see that you're leveraging that here. You could make this a bit more dynamic, and not hardcode the id in this test by doing something like:

```ruby
quotes= QuoteFacade.your_method(keyword)[0]
quotes.each do |quote|
   expect(page).to have_content(quote.content)
   # .. and so on...
end 
```